src/frontend/typer/søknad.ts

```diff
@@ -95,8 +95,8 @@ export interface ISøknadKontraktSøker {
     sivilstand: ISøknadsfelt<ESivilstand>;
     spørsmål: SpørsmålMap;
     utvidet?: ISøknadsfelt<{
-        tidligereSamboere: IKontraktTidligereSamboer[];
-        nåværendeSamboer: IKontraktNåværendeSamboer | null;
+        tidligereSamboere: ISøknadsfelt<IKontraktTidligereSamboer>[];
+        nåværendeSamboer: ISøknadsfelt<IKontraktNåværendeSamboer> | null;
         spørsmål: SpørsmålMap;
```

Comment:
Underveis i testinga av mapping fra ISøknad til ISøknadKontrakt så dukka det opp noe som ikke stemte med typene til tidligereSamboer og nåværendeSamboer. javascripten gjør rett og er ihht kontrakten, men typescript var ikke det. Manglet ISøknadsfelt for begge. Hvorfor typescript ikke sa ifra at dette ikke stemte vet jeg ikke. Men ved å sette på typer på søknadsfelt funksjonen så sa den i fra.